src/Microsoft.DotNet.Configurer/xlf/LocalizableStrings.cs.xlf

```diff
@@ -45,7 +45,7 @@ Běží příkaz pro naplnění vaší místní mezipaměti balíčků, aby se v
 Here are some options to fix this error:
 ----------------------------------------
 1. Re-run this command with elevated access.
-2. Disabled the first run experience by setting the environment variable DOTNET_SKIP_FIRST_TIME_EXPERIENCE to true.
+2. Disable the first run experience by setting the environment variable DOTNET_SKIP_FIRST_TIME_EXPERIENCE to true.
```

Comment:
This should not be replaced manually, it should be updated by xliff-tasks, and the state should have changed to needs-review-translation. Technically, the translations probably wouldn't be impacted by an English grammar change, but we shouldn't assume that. I've pushed a commit to fix this up.